content/blog/pulumi-components/index.md

```diff
@@ -0,0 +1,121 @@
+---
+
+Pulumi Components enable you to create, share, and consume reusable infrastructure building blocks across your organization and the broader community. Today, we're excited to announce significant enhancements to Pulumi Components that make them more powerful, accessible, and easier to use than ever before.
+
+<!--more-->
+
+With this release, we've made it possible to author components in one language and consume them in any other Pulumi language—including Pulumi YAML. This breakthrough enables platform teams to build sophisticated infrastructure abstractions in their preferred programming language while application developers can easily consume these components using simpler interfaces or even YAML, all without sacrificing type safety or functionality.
+
+For platform teams, the new Pulumi Components offer a more intuitive, more secure way to codify organizational standards. For developers, it's a straightforward way to pull in rock-solid abstractions without having to worry about all the underlying details. It's a win-win that drastically accelerates innovation and drives productivity across your teams.
+
+"Pulumi's reusable components have transformed our infrastructure collaboration," said Kevin Keeler, Vice President - DevOps, QA, and Architecture at A+E Networks. "Our developers interact with Pulumi YAML to provision infrastructure quickly and easily, while our platform team leverages Pulumi's programming capabilities to build robust, reusable components. This ensures compliance with organizational standards and best practices without burdening developers with complexity. By streamlining this workflow, we've enhanced productivity and focused more on delivering customer value."
+
+## Understanding Component Resources
+
+At their core, Component Resources in Pulumi are logical groupings of resources that encapsulate infrastructure patterns and best practices. Unlike standard resources that map directly to cloud provider resources, component resources are higher-level abstractions that can contain multiple child resources working together to implement a specific capability. While similar in concept to Terraform modules, Pulumi Components offer significantly more power through full programming language capabilities, stronger typing, inheritance patterns, and now, cross-language support.
+
+For example, a `WebService` component might include a load balancer, compute instances, security groups, and networking configuration—all bundled together as a single, reusable unit.
+
+## What's New with Pulumi Components
```

Comment:
A high-level comment on the blog: I have a sense that we aren't crisply saying what exactly is new. Git sources are covered. But not the magic of deriving the schema from an existing component class and exposing it through the machinery of providers? It's a tricky balance to explain this simply but still give the appreciation. Or do we want a follow-up blog post that explains that more deeply?

Reply:
Mikhail, I did think a variant of the "table" you showed before would be helpful. E.g., that you can still manually author schema if you want, and/or pre-compile it (for Go, etc), and/or publish to package managers, and/or ...

Basically, make sure folks know the building blocks are still there, we just made the golden path of how to assemble them much easier.